python/llm/example/transformers/transformers_int4/chatglm/README.md

```diff
@@ -2,13 +2,17 @@
 
 In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on ChatGLM models. For illustration purposes, we utilize the [THUDM/chatglm-6b](https://huggingface.co/THUDM/chatglm-6b) as a reference ChatGLM model.
 
-## 0. Requirements
+> **Note**: If you want to download the model, please refer to [here](https://huggingface.co/docs/hub/models-downloading#using-git). And you don't need to convert the original [THUDM/chatglm-6b](https://huggingface.co/THUDM/chatglm-6b) model.
+
+## Requirements
 To run these examples with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
 
 ## Example: Predict Tokens using `generate()` API
 In the example [generate.py](./generate.py), we show a basic use case for a ChatGLM model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
 ### 1. Install
-We suggest using conda to manage environment:
+We suggest using conda to manage the python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a python environment for BigDL-LLM:
```

Comment:
conda -> Conda
python -> Python
And maybe emphasize that we suggest Python 3.9 here :)